account_banking_pain_base/post_install.py

```diff
@@ -3,13 +3,13 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 
-from openerp import SUPERUSER_ID
+from odoo import api, SUPERUSER_ID
 
 
-def set_default_initiating_party(cr, pool):
-    company_ids = pool['res.company'].search(cr, SUPERUSER_ID, [])
-    companies = pool['res.company'].browse(cr, SUPERUSER_ID, company_ids)
-    for company in companies:
-        pool['res.company']._default_initiating_party(
-            cr, SUPERUSER_ID, company)
+def set_default_initiating_party(cr, registry):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        companies = env['res.company'].search([])
+        for company in companies:
+            env['res.company']._default_initiating_party(company)
```

Comment:
The `_default_initiating_party` method should be converted from `@api.model` to `@api.multi` without the *self recordset* argument, to follow v10.0 standard changes.
Then, you will be able to directly call `companies._default_initiating_party()`.